### ISSUE TITLE
Docs: Update manual install in README to include gnome-shell-extension-prefs

### ISSUE THREAD

**Issue by allella** [mzur/gnome-shell-wsmatrix#205]
It look me a decent amount of time to hunt down the preferences in Gnome 40 after doing a manual .zip install.  I just don't like installing browser extensions and running daemons, which seems to be the default "easy" method.

Also, I forgot I used Gnome Tweaks with earlier version to edit the Workspace preferences, but I see Tweaks is no longer suggested tool and the new way is:

`gnome-shell-extension-prefs`
or
`gnome-extensions prefs wsmatrix@martin.zurowietz.de`

Would you take a PR if I update the README to include notes on opening the preferences?

Also, after unzipping Gnome didn't recognize the newly unzipped extension until after I restarted. Only then would `gnome-extensions enable wsmatrix@martin.zurowietz.de` recognize the extension was available. The README mentioned needing to restart only after enabling, so I'm guessing two restarts may now be needed since Gnome 40?

In any case, if you'd like me to submit a PR to address either, or both, of these suggestions, then please confirm or let me know.

Thanks

**Comment by mzur** [comment]
Yes please send a PR, thanks! I have the following comments:
- Did you restart your computer or just GNOME? I think the latter should do. So it would be "restart+enable+restart", basically.
- Please add the part about `extension-prefs` to the FAQ section, as this is not only relevant for the manual install.

**Comment by allella** [comment]
Alright, I'll create a PR.

I restarted the computer because I was also testing Thunderbolt Dock stability, but I can document the Gnome Shell restart, extension enable, Gnome Shell restart.

It looks like gnome-shell-extension-prefs is deprecated, so I'll note using
`gnome-extensions prefs wsmatrix@martin.zurowietz.de`

Has `gnome-extension prefs` existed since Gnome 3.28, when this fork was started? I found the release notes, but I'm not clear on the history of the project and don't want to document this as "the way" if it only works for never versions.

Thanks
